test/compose/test-compose

```diff
@@ -247,6 +247,17 @@ function podman() {
     return $rc
 }
 
+# as rootless we want to test the remote connection so we add --connection
+function podman_compose() {
+    if is_rootless; then
+        $PODMAN_BIN --connection compose-sock compose "$@"
+    else
+        podman compose "$@"
+    fi
+    rc=$?
+    return $rc
```

Comment:
These 2 lines should not be necessary. Function return value is the result of the last executed command.